### PR TITLE
add log

### PR DESCRIPTION
**Pull request by DanLiu2Intel**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Comment by DanLiu2Intel** [comment]
not add -iol

./compile_tool -m ../testConfigandData/googlenet-v1.xml -d NPU  
```
(base) npu@npu-mtl-24:~/test/bin_testmlir$ ./compile_tool -m ../testConfigandData/googlenet-v1.xml -d NPU
OpenVINO Runtime version ......... 2025.1.0
Build ........... 2025.1.0-17965-8d5f583bc7e
Parsing command-line arguments
Checking FLAGS_LOG_LEVEL
Reading model
Performing reshape
Configuring model pre & post processing
      configurePrePostProcessing_366
      ================Network inputs:
      ==1=> in name: input ,
      ==1=> in name: input , input:0 ,
      ==2=> in name: input , input:0
      ==3=> node_name: input
         =========== input , input:0 (node: input) : f32 / [...] / [1,3,224,224]
      =====================Network outputs:
      ==1=> out name: InceptionV1/Logits/Predictions/Softmax ,
      ==1=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 ,
      ==2=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0
      ======3====>out node name: InceptionV1/Logits/Predictions/Softmax
          ===========InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 (node: InceptionV1/Logits/Predictions/Softmax) : f32 / [...] / [1,1001]
      
      configurePrePostProcessing_369
      ================Network inputs:
      ==1=> in name: input ,
      ==1=> in name: input , input:0 ,
      ==2=> in name: input , input:0
      ==3=> node_name: input
         =========== input , input:0 (node: input) : f32 / [...] / [1,3,224,224]
      =====================Network outputs:
      ==1=> out name: InceptionV1/Logits/Predictions/Softmax ,
      ==1=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 ,
      ==2=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0
      ======3====>out node name: InceptionV1/Logits/Predictions/Softmax
          ===========InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 (node: InceptionV1/Logits/Predictions/Softmax) : f32 / [...] / [1,1001]

Printing Input and Output Info from model
Network inputs:
    input : f32 / [...] / [1,3,224,224]
Network outputs:
    InceptionV1/Logits/Predictions/Softmax:0 : f32 / [...] / [1,1001]

Parsing configuration file
Compiling model
[WARNING] 18:53:25.584 [NPUZeroInitStructsHolder] Some features might not be available! Plugin L0 API minor version = 12, Driver L0 API minor version = 11
-----------DriverCompilerAdapter::compile1--------------
      ===from compile_tool=====Network inputs:
          input : f32 / [...] / [1,3,224,224]
      Network outputs:
          InceptionV1/Logits/Predictions/Softmax:0 : f32 / [...] / [1,1001]
-----------DriverCompilerAdapter::compile2--------------
      =======from benchmark=========Network inputs:
      ==1=> in name: input ,
      ==1=> in name: input , input:0 ,
      ==2=> in name: input , input:0
      ==3=> node_name: input
         =========== input , input:0 (node: input) : f32 / [...] / [1,3,224,224]
      =====================Network outputs:
      ==1=> out name: InceptionV1/Logits/Predictions/Softmax ,
      ==1=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 ,
      ==2=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0
      ======3====>out node name: InceptionV1/Logits/Predictions/Softmax
          ===========InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 (node: InceptionV1/Logits/Predictions/Softmax) : f32 / [...] / [1,1001]
-----------DriverCompilerAdapter::compile3--------------
--------- DriverCompilerAdapter::serializeIOInfo flag is --inputs_precisions="input:FP32" --inputs_layouts="input:NCHW" --outputs_precisions="InceptionV1/Logits/Predictions/Softmax                          :FP32" --outputs_layouts="InceptionV1/Logits/Predictions/Softmax:NC"
[WARNING] 18:53:25.601 [serializeConfig] COMPILER_DYNAMIC_QUANTIZATION property is not supported by this compiler version. Removing from parameters
Writing into file - googlenet-v1.blob
Done. LoadNetwork time elapsed: 510 ms
```

**Comment by DanLiu2Intel** [comment]
-iol "input:NCHW,InceptionV1/Logits/Predictions/Softmax:NC" layout node name has been change....
./compile_tool -m ../testConfigandData/googlenet-v1.xml -d NPU -iol "input:NCHW,InceptionV1/Logits/Predictions/Softmax:NC"

```
(base) npu@npu-mtl-24:~/test/bin_testmlir$ ./compile_tool -m ../testConfigandData/googlenet-v1.xml -d NPU -iol "input:NCHW,InceptionV1/Logits/Predictions/Softmax:NC"
OpenVINO Runtime version ......... 2025.1.0
Build ........... 2025.1.0-17965-8d5f583bc7e
Parsing command-line arguments
Checking FLAGS_LOG_LEVEL
Reading model
Performing reshape
Configuring model pre & post processing
configurePrePostProcessing_366
        ================Network inputs:
        ==1=> in name: input ,
        ==1=> in name: input , input:0 ,
        ==2=> in name: input , input:0
        ==3=> node_name: input
           =========== input , input:0 (node: input) : f32 / [...] / [1,3,224,224]
        =====================Network outputs:
        ==1=> out name: InceptionV1/Logits/Predictions/Softmax ,
        ==1=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 ,
        ==2=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0
        ======3====>out node name: InceptionV1/Logits/Predictions/Softmax
            ===========InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 (node: InceptionV1/Logits/Predictions/Softmax) : f32 / [...] / [1,1001]
configurePrePostProcessing_369
        ================Network inputs:
        ==1=> in name: input ,
        ==1=> in name: input , input:0 ,
        ==2=> in name: input , input:0
        ==3=> node_name: input
           =========== input , input:0 (node: input) : f32 / [N,C,H,W] / [1,3,224,224]
        =====================Network outputs:
        ==1=> out name: InceptionV1/Logits/Predictions/Softmax ,
        ==1=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 ,
        ==2=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0
        ======3====>out node name: Convert_1823
            ===========InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 (node: Convert_1823) : f32 / [N,C] / [1,1001]
Printing Input and Output Info from model
        Network inputs:
            input : f32 / [N,C,H,W] / [1,3,224,224]
        Network outputs:
            InceptionV1/Logits/Predictions/Softmax:0 : f32 / [N,C] / [1,1001]
Parsing configuration file
Compiling model
[WARNING] 18:59:32.191 [NPUZeroInitStructsHolder] Some features might not be available! Plugin L0 API minor version = 12, Driver L0 API minor version = 11
-----------DriverCompilerAdapter::compile1--------------
===from compile_tool=====Network inputs:
        input : f32 / [N,C,H,W] / [1,3,224,224]
    Network outputs:
        InceptionV1/Logits/Predictions/Softmax:0 : f32 / [N,C] / [1,1001]
-----------DriverCompilerAdapter::compile2--------------
=======from benchmark=========Network inputs:
        ==1=> in name: input ,
        ==1=> in name: input , input:0 ,
        ==2=> in name: input , input:0
        ==3=> node_name: input
           =========== input , input:0 (node: input) : f32 / [N,C,H,W] / [1,3,224,224]
        =====================Network outputs:
        ==1=> out name: InceptionV1/Logits/Predictions/Softmax ,
        ==1=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 ,
        ==2=> out name: InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0
        ======3====>out node name: Convert_1823
            ===========InceptionV1/Logits/Predictions/Softmax , InceptionV1/Logits/Predictions/Softmax:0 (node: Convert_1823) : f32 / [N,C] / [1,1001]
-----------DriverCompilerAdapter::compile3--------------
--------- DriverCompilerAdapter::serializeIOInfo flag is --inputs_precisions="input:FP32" --inputs_layouts="input:NCHW" --outputs_precisions="Convert_1823:FP32" --outputs_layouts="Convert_1823:NC"
[WARNING] 18:59:32.206 [serializeConfig] COMPILER_DYNAMIC_QUANTIZATION property is not supported by this compiler version. Removing from parameters
Writing into file - googlenet-v1.blob
Done. LoadNetwork time elapsed: 487 ms
```

**Comment by DanLiu2Intel** [comment]
ov:
DriverCompilerAdapter::compile(){
    buildFlags += serializeIOInfo(model, useIndices);
    buildFlags += " ";
    buildFlags += serializeConfig(config, compilerVersion);
    ...... 
    ze_graph_handle_t graphHandle = _zeGraphExt->getGraphHandle(std::move(serializedIR), buildFlags, flags); 
///return typedef struct _ze_graph_handle_t *ze_graph_handle_t;  (levelzero repo typedef)
///src/plugins/intel_npu/src/compiler_adapter/include/ze_graph_ext_wrappers.hpp:25 (Adapter to use CiD through ZeroAPI)
///src/plugins/intel_npu/src/compiler_adapter/src/ze_graph_ext_wrappers.cpp:327 368
/////_zeroInitStruct->getGraphDdiTable().pfnCreate() or _zeroInitStruct->getGraphDdiTable().pfnCreate2()
////////    inline ze_graph_dditable_ext_curr_t& getGraphDdiTable() const {  return *graph_dditable_ext_decorator; }
///////     using ze_graph_dditable_ext_curr_t = ze_graph_dditable_ext_decorator;
//////////////src/plugins/intel_npu/src/utils/include/intel_npu/utils/zero/zero_types.hpp:29
/////////////////src/plugins/intel_npu/thirdparty/level-zero-ext/ze_graph_ext.h:243 
 ```
  typedef ze_result_t (ZE_APICALL *ze_pfnGraphCreate_ext_t)(///////////
      ze_context_handle_t hContext,                   ///< [in] handle of the context
      ze_device_handle_t hDevice,                     ///< [in] handle of the device
      const ze_graph_desc_t* desc,                    ///< [in] pointer to graph descriptor
      ze_graph_handle_t* phGraph                      ///< [out] pointer to handle of graph object created
      );
  ```
Q: graph_dditable_ext_t is from where? firmware?
}

kmb:
src/vpux_driver_compiler/src/vpux_compiler_l0/vcl_bridge.cpp:172 vclExecutableCreate call 
  src/vpux_driver_compiler/src/vpux_compiler_l0/vcl_compiler.cpp:222  importNetwork
  src/vpux_driver_compiler/src/vpux_compiler_l0/vcl_compiler.cpp:302  importNetwork
